R/pkg/R/generics.R

```diff
@@ -1279,6 +1279,19 @@ setGeneric("spark.naiveBayes", function(data, formula, ...) { standardGeneric("s
 #' @export
 setGeneric("spark.survreg", function(data, formula, ...) { standardGeneric("spark.survreg") })
 
+#' @rdname spark.lda
+#' @export
+setGeneric("spark.lda", function(data, ...) { standardGeneric("spark.lda") })
```

Comment:
Consider whether we need to have `...` in the signature. If it's present we need to add `@param ... description` otherwise CRAN check will flag this.
I think if it is not needed we should remove it.

Same for all other generic being added.